boilerplate/webpack.config.js

```diff
@@ -9,10 +9,18 @@ const config = {
       {
         test: /\.elm$/,
         exclude: [/elm-stuff/, /node_modules/],
-        use: ["elm-webpack-loader"]
+        use: [
+          {
+            loader: "elm-webpack-loader",
+            options: {
+              forceWatch: isDev,
```

Comment:
elm-webpack-loader detects weather webpack is in watch mode or not by checking argv arguments: https://github.com/elm-community/elm-webpack-loader/blob/25e7588dfbb789f6d30de91f0bd395aed1e04db7/index.js#L67

this breaks watch for use because we are using webpack programatically, not via CLI, I think elm-webpack-loader should not relly on that, but meanwhile, forceWatch fixes it